ros2topic/ros2topic/verb/echo.py

```diff
@@ -69,7 +69,7 @@ def add_arguments(self, parser, cli_name):
             )
         )
         parser.add_argument(
-            '--field', type=str, default=None,
+            '--field', action='append', type=str, default=None,
```

Comment:
(just a comment) probably we would want to change this into `--fields` but that breaks the user environment so lets keep `--field`